syllabus/4-Smart_Contracts/6-Wasm_ink!_slides.md

```diff
@@ -288,26 +286,17 @@ cargo contract instantiate --constructor default --suri //Alice
 
 Output:
 
-<div style="font-size: 0.82em;">
+<!-- <div style="font-size: 0.82em;"> -->
```

Comment:
please remove commented part if not needed

Reply:
done